src/main/java/org/galatea/starter/service/HalService.java

```diff
@@ -0,0 +1,120 @@
+package org.galatea.starter.service;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+import net.sf.aspect4log.Log;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@ToString
+@EqualsAndHashCode
+@Slf4j
+@Log
+@Service
+public class HalService {
```

Comment:
for now we can just have empty methods that either return an empty string or nothing. as we implement the real functionality we will fill those methods.

Reply:
- [ ] Return empty strings for methods without final implementation